soc/arm/nxp_imx/rt5xx/soc.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2022, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief System/hardware module for NXP RT5XX platform
+ *
+ * This module provides routines to initialize and support board-level
+ * hardware for the RT5XX platforms.
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <soc.h>
+#include <drivers/uart.h>
+#include <linker/sections.h>
+#include <arch/cpu.h>
+#include <aarch32/cortex_m/exc.h>
+#include <fsl_clock.h>
+#include <fsl_common.h>
+#include <fsl_device_registers.h>
+#include <clock_config.h>
```

Comment:
please review include list here, e.g. why `drivers/uart.h` here?

Reply:
Thanks for pointing this, I have trimmed this list of includes.